packages/ra-ui-materialui/src/button/SaveButton.tsx

```diff
@@ -34,19 +32,11 @@ import { FormRenderProps } from 'react-final-form';
  * @prop {string|boolean} redirect Override of the default redirect in case of success. Can be 'list', 'show', 'edit' (for create views), or false (to stay on the creation form).
  * @prop {function} onSave (deprecated)
  * @prop {function} onSuccess Callback to execute instead of the default success side effects. Receives the dataProvider response as argument.
- * @prop {function} onFailure Callback to execute instead of the default error side effects. Receives the dataProvider error response as argument.
```

Comment:
this is incorrect: SaveButton now only receives mutationOptions

Reply:
Both onFailure and onSuccess have been replaced with mutationOptions